src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Jobs/GetAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -167,8 +176,12 @@ public override void ExecuteCmdlet()
                 int resultCount = 0;
                 if (UseSecondaryRegion.IsPresent)
                 {
-                    ARSVault vault = ServiceClientAdapter.GetVault(resourceGroupName, vaultName);
-                    string secondaryRegion = BackupUtils.regionMap[vault.Location];
+                    if (VaultLocation == null || VaultLocation == "")
```

Comment:
VaultLocation already had attribute [ValidateNotNullOrEmpty]. Could this if condition be true?